pkg/ebpf/tracer_windows.go

```diff
@@ -119,26 +120,26 @@ func makeDDAPIVersionBuffer(signature uint64) []byte {
 }
 
 // GetActiveConnections returns all active connections
-func (t *Tracer) GetActiveConnections(_ string) (*Connections, error) {
-	return &Connections{
+func (t *Tracer) GetActiveConnections(_ string) (*network.Connections, error) {
```

Comment:
Is there any reason to leave `tracer.go` in the ebpf package?  It seems wrong to have windows related code stay in the ebpf package.

Reply:
At some point I'd like to move that too, but right now the effort would be huge since there is a lot of coupling between the tracer code and a bunch of eBPF private utilities/helpers, so I decided to leave it for now.